openfga/v1/openfga_service.proto

```diff
@@ -1183,7 +1183,9 @@ message ExpandRequest {
     (google.api.field_behavior) = REQUIRED
   ];
 
-  string authorization_model_id = 3 [
+  openfga.v1.ContextualTupleKeys contextual_tuples = 3 [json_name = "contextual_tuples"];
```

Comment:
Please do not alter the number of the existing fields (see https://github.com/openfga/api/pull/202/files)

Please put the new field at the end of this list with value `5`

Reply:
Noted, the changes have been made. 👍